frontend/app/src/store/balances/blockchain-accounts.ts

```diff
@@ -1083,7 +1096,10 @@ export const useBlockchainAccountsStore = defineStore(
 
     const ethDetectedTokensRecord = ref<EthDetectedTokensRecord>({});
 
-    const fetchDetectedTokens = async (address?: string) => {
+    const fetchDetectedTokens = async (
+      address: string | null,
+      fetchBalances: boolean = true
```

Comment:
wouldn't it make sense to split the balance fetching again in a new action?

Reply:
Hmm, I don't really understand, can you explain a bit?

Reply:
Ideally, I was thinking of having distinct actions for each task.

1. Adding account
2. Detecting tokens
3. Fetching balances


And for anything complicated, we do an action that calls the ones above.

Reply:
I will take out the refresh balances from detecting tokens.

But for the add account, previously it also call `updateBlockchainBalances` directly inside, and since fetching tokens should be called before `updateBlockchainBalances`, doesn't it makes sense to keep calling the detect tokens inside addAccount? @kelsos

Reply:
Since we are in a hurry we can keep that at the moment and I can spend some time thinking about the new approach.